ios/native/NativeProxy.mm

```diff
@@ -27,6 +30,41 @@
 using namespace facebook;
 using namespace react;
 
+static CGFloat SimAnimationDragCoefficient(void)
```

Comment:
The compiler should do it by itself but to be sure you can use `static inline` for both definitions.

Reply:
In fact, `inline` is just a hint. I wanted to leave it to the compiler since it should trivially optimize this function call (preprocessor macro leaves just `return NULL ? <unreachable> : 1.f` and this function is used only in this translation unit).